src/server/capacity_unit_calculator.cpp

```diff
@@ -93,25 +93,36 @@ void capacity_unit_calculator::add_get_cu(int32_t status,
                                           const dsn::blob &value)
 {
     _pfc_get_bytes->add(key.size() + value.size());
-    if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound) {
+    if (status == rocksdb::Status::kNotFound) {
+        add_read_cu(1);
+        return;
+    }
+
+    if (status != rocksdb::Status::kOk) {
         return;
     }
-    add_read_cu(value.size());
+    add_read_cu(key.size() + value.size());
 }
 
 void capacity_unit_calculator::add_multi_get_cu(int32_t status,
                                                 const dsn::blob &hash_key,
                                                 const std::vector<::dsn::apps::key_value> &kvs)
 {
     int64_t data_size = 0;
+    int64_t multi_get_bytes = 0;
     for (const auto &kv : kvs) {
-        data_size += kv.key.size() + kv.value.size();
+        multi_get_bytes += kv.key.size() + kv.value.size();
+        data_size += hash_key.size() + kv.key.size() + kv.value.size();
```

Comment:
Why add `hash_key.size()` multiple times?

Reply:
@neverchanje suggest it and i think `multi_put` equal `put` n “hash_key:sort_key:value”, though the `hash_key` is same.

Reply:
Could you please paste a link to how other public cloud service calculate CU? do they sum up all the 'hash key size'?

Reply:
> Based on the aliyun([operation cu definition of aliyun](https://help.aliyun.com/document_detail/27304.html?spm=a2c4g.11186623.6.742.451d3354kD2Rb2), [cu calculate example](https://help.aliyun.com/document_detail/27298.html?spm=5176.11065259.1996646101.searchclickresult.41a62ac2iYd8qG)), the way of pegasus to calculate cu is inaccurate that we sometimes ignore the key cu.

As description of pr